sys/fido2/ctap/transport/Kconfig

```diff
@@ -4,4 +4,9 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
+menuconfig MODULE_FIDO2_CTAP_TRANSPORT
```

Comment:
Is it valid to have no transport?

Reply:
Yes it is. The authenticator can be internal.

Reply:
All right!